mods/persistence/controllers/subsystems/mining.dm

```diff
@@ -53,32 +54,43 @@ SUBSYSTEM_DEF(mining)
 
 	var/regen_interval = 90		// How often in minutes to generate mining levels.
```

Comment:
The mines are hardly being emptied as it stands given reports, and given the lag caused at least triple the time between regen's is valid.  If they dry up, an admin can manually regen them.
`var/regen_interval = 270`       // How often in minutes to generate mining levels.